### PR TITLE
Idle timeout for blocking http

### PR DESCRIPTION
**Pull request by michalszynkiewicz**
refs #4455

### CODE REVIEW

**Comment by gsmet** [comment]
This uses a SNAPSHOT dependency, it shouldn't be merged.

**Comment by michalszynkiewicz** [comment]
requires https://github.com/quarkusio/quarkus-http/pull/10

**Comment by gsmet** [comment]
I force pushed the Beta3 version change as it has been pushed to Central.

let's see what CI has to say.

**Comment by michalszynkiewicz** [comment]
@gsmet  the released version does not contain https://github.com/quarkusio/quarkus-http/pull/10 so the tests will fail

File: test-framework/common/src/main/java/io/quarkus/test/common/RestAssuredURLManager.java [hunk 1]
```diff
@@ -92,6 +91,15 @@ public void setURL() {
         }
     }
 
+    public static String getHost() {
```
Comment (stuartwdouglas) [inline test-framework/common/src/main/java/io/quarkus/test/common/RestAssuredURLManager.java]: You should not need these changes. Just inject the URL using @TestHTTPResource and then get the host and port from the URL.

**Review by stuartwdouglas — CHANGES_REQUESTED** [review]
Can we get a version of this with just the vert.x idle timeout changes for the 28.0 release?

**Comment by michalszynkiewicz** [comment]
@stuartwdouglas uploaded setting the timeout only

**Review by stuartwdouglas — APPROVED** [review]

**Pull request merged**